packages/tupaia-web/src/utils/getApiUrl.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export const getApiUrl = () => {
```

Comment:
I think that in lesmis there was a bit more going on to get the api url but here it is very simple and also probably only ever used in the api function so can we please move this function in to the api.ts file.